clients/src/main/java/org/apache/kafka/common/metrics/Metrics.java

```diff
@@ -130,6 +139,27 @@ public Thread newThread(Runnable runnable) {
         } else {
             this.metricsScheduler = null;
         }
+
+        addMetric(new MetricName("count", "kafka-metrics-count", "total number of registered metrics", defaultConfig.tags()),
+          new MeasurableStat() {
+            @Override
+            public double measure(MetricConfig config, long now) {
+              return metrics.size();
+            }
+
+            @Override
+            public void record(MetricConfig config, double value, long timeMs) {
+
+            }
+          });
+    }
```

Comment:
Instead of using MeasurableStat(), it seems that we can just use Measurable like the following.

new Measurable() {
public double measure(MetricConfig config, long now) {
return metrics.size();
}